src/main/resources/api.yaml

```diff
@@ -490,13 +490,126 @@ paths:
           description: User do not have permissions to get quiz binding
         '404':
           description: Quiz not found
+  /permissions/resources/{resourceId}:
+    post:
+      summary: add permission to the resource
+      operationId: setPermissionToResource
+      tags:
+        - permissions
+      parameters:
+        - name: resourceId
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: "sample00-id-0000-0000-quiz00000000"
+            description: "UUID of the resource"
+      requestBody:
+        description: Permission to add
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PermissionDto'
+      responses:
+        '200':
+          description: Permission added to resource
+          #todo co my chcemy zwórcić?
+        '400':
+          description: Bad request
+        '403':
+          description: User do not have permissions to set permission to the resource
+        '404':
+          description: Resource not found
+    put:
+      summary: edit permission to the resource
+      operationId: editPermissionToResource
+      tags:
+        - permissions
+      parameters:
+        - name: resourceId
+          in: path
+          required: true
+          schema:
+            type: string
+            format: uuid
+            example: "sample00-id-0000-0000-quiz00000000"
+            description: "UUID of the resource"
+      requestBody:
+        description: Permission to edit
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/PermissionDto'
+      responses:
+        '200':
+          description: Permission edited to resource
```

Comment:
in that case it's no obvious because we create only a conncetion which isnt typical object. I may return it but it do not seems to be as necessary as f.e. created new note